string source
time stamp
uint32 removedTermId
memory/Term addedOrRemovedTerm
